ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -87,6 +87,15 @@ func rayClusterTemplate(name string, namespace string) *rayv1.RayCluster {
 	}
 }
 
+func updateAllPodsToRunning(ctx context.Context, podList corev1.PodList, podFilters []client.ListOption, podLabel string) {
```

Comment:
```suggestion
func updateAllPodsToRunning(ctx context.Context, podList corev1.PodList, message string) {
```
It is wired to have `podList` and `podFilters` parameters at the same time. We should only keep one of them.